### DISCUSSION TITLE
Bindings support

### DISCUSSION THREAD

**Discussion by kassane** [tursodatabase/libsql#14] [General]
SQLite already has several bindings supported by other languages.

**e.g.**

- Rust: [rusqlite](https://github.com/rusqlite/rusqlite)
- Go: [go-sqlite3](https://github.com/mattn/go-sqlite3)
- Zig: [zig-sqlite](https://github.com/vrischmann/zig-sqlite)
- Python3: [src/libsqlite3](https://github.com/python/cpython/tree/3.10/Lib/sqlite3/)
- ...

Would you be able to clarify what would be the strengths for the transition from SQLite to libSQL?
What would the compatibility level be?

**Comment by penberg** [comment]
@kassane We will aim to keep libSQL a drop-in replacement (ABI compatible) with SQLite so there's minimal effort needed to transition to libSQL if they need the new functionality in libSQL, for example.
